docs_user/modules/proc_deploying-backend-services.adoc

```diff
@@ -297,6 +313,7 @@ spec:
   ovn:
     enabled: false
     template:
+ifeval::["{OpenStackPreviousInstaller}" == "director"]
```

Comment:
@pinikomarov Shouldn't this change to:` ifeval::["{OpenStackPreviousInstaller}" != "director_operator"] ` ?

Reply:
changed